test-infra/auto-deploy/Dockerfile

```diff
@@ -0,0 +1,165 @@
+# Docker image for nightly deployment cronjob.
```

Comment:
This dockerfile looks like its duplicating a lot of our test-worker? Is that deliberate?